modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/message/SchedulerUpdateHandler.java

```diff
@@ -79,8 +79,7 @@ else if (BooleanUtils.toBoolean(publishLive)) {
           break;
         case Delete:
         case DeleteRecordingStatus:
-          if (isLive(mpId))
-            liveScheduleService.deleteLiveEvent(mpId);
+          liveScheduleService.deleteLiveEvent(mpId);
```

Comment:
It seems that omitting the isLive check here will cause the live schedule service to issue a search query to find the media package mpId and then does another test (based on tracks) to find out whether it is live.

Can you please provide some background information about this change?

Maybe a short comment that explain why at this particular code location, isLive is not used?

Reply:
When a delete notification comes in, the workflow configuration, which is checked by "isLive" to determine if this is a live event, does not exist anymore. This is actually a bug fix :)

Reply:
Thanks for clarifying this.